app/Exceptions/Handler.php

```diff
@@ -39,6 +39,10 @@ class Handler extends ExceptionHandler
      */
     public function report(Exception $exception)
     {
+        if (app()->bound('sentry') ) {
```

Comment:
I was wondering why did you not include `&& $this->shouldReport($exception)` to the if statement?

Reply:
When I made some testing I noticed that $this->shouldReport did not return anything for error 401 so I thought we needed to define shouldReport through some middleware.

After further testing I believe that when an exception should NOT be reported shouldReport will return nothing (which evaluates as false) and shouldntReport will return 1 (which evaluates as true)

Screenshot of logs for exception 401:

<img width="682" alt="screen shot 2018-09-10 at 11 32 33" src="https://user-images.githubusercontent.com/25614957/45292507-591b2480-b4ed-11e8-9243-714411dc368e.png">

<img width="553" alt="screen shot 2018-09-10 at 11 30 32" src="https://user-images.githubusercontent.com/25614957/45292492-50c2e980-b4ed-11e8-90da-9271fea8c4e7.png">